pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1528 @@
+        return self._dll_interface.string_to_enum(FilterClass, type_string)
+
+    @filter_class.setter
+    def filter_class(self, filter_class: FilterClass):
+        if filter_class:
+            string_value = self._dll_interface.enum_to_string(filter_class)
+            self._dll_interface.set_string(self._dll.setFilterClass, string_value)
+
+    @property
+    def filter_implementation(self) -> FilterImplementation:
+        """The technology used to implement the filter.
+        The full list of implementations are listed in FilterImplementation enum.
+        The default is `LUMPED`.
+
+        Returns
+        -------
+        :enum:'FilterImplementation'
+        """
+        type_string = self._dll_interface.get_string(self._dll.getFilterImplementation)
+        return self._dll_interface.string_to_enum(FilterImplementation, type_string)
+
+    @filter_implementation.setter
+    def filter_implementation(self, filter_implementation: FilterImplementation):
+        if filter_implementation:
+            string_value = self._dll_interface.enum_to_string(filter_implementation)
+            self._dll_interface.set_string(self._dll.setFilterImplementation, string_value)
+
+    @property
+    def diplexer_type(self) -> DiplexerType:
+        """The type of diplexer topology. Only applicable to lumped filters.
+        The full list of diplexer types are listed in DiplexerType enum.
+        The default is `HI_LO` for `DIPLEXER_1` filter class.
+        The default is `BP_BS` for `DIPLEXER_2` filter class.
+
+        Returns
+        -------
+        :enum:`DiplexerType`
+        """
+        type_string = self._dll_interface.get_string(self._dll.getDiplexerType)
+        return self._dll_interface.string_to_enum(DiplexerType, type_string)
+
+    @diplexer_type.setter
+    def diplexer_type(self, diplexer_type: DiplexerType):
+        string_value = self._dll_interface.enum_to_string(diplexer_type)
+        self._dll_interface.set_string(self._dll.setDiplexerType, string_value)
+
+    @property
+    def filter_multiple_bands_enabled(self) -> bool:
+        """Whether to enable the multiple bands table. The default is `False`.
+
+        Returns
+        -------
+        bool
+        """
+        filter_multiple_bands_enabled = c_bool()
+        status = self._dll.getMultipleBandsEnabled(byref(filter_multiple_bands_enabled))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(filter_multiple_bands_enabled.value)
+
+    @filter_multiple_bands_enabled.setter
+    def filter_multiple_bands_enabled(self, filter_multiple_bands_enabled: bool):
+        status = self._dll.setMultipleBandsEnabled(filter_multiple_bands_enabled)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def filter_multiple_bands_low_pass_frequency(self) -> str:
+        """The multiple bands low pass frequency of combined low pass and band pass filters. The default is `1GHz`.
```

Comment:
```suggestion
        """Multiple bands low-pass frequency of combined low-pass and band-pass filters. The default is ``1GHz``.
```
There seems like there has to be a better way to write the description for this property! I really have no idea what this says!